src/NLog/Logger.cs

```diff
@@ -123,21 +133,20 @@ public Logger WithProperty(string propertyKey, object propertyValue)
         /// </remarks>
         /// <param name="propertyKey">Property Name</param>
         /// <param name="propertyValue">Property Value</param>
+        [Obsolete("Instead use the Properties-property to access or modify the active collection")]
```

Comment:
I removed the obsolete, as I have some issues with an obsolete so soon after introduction. 

also changed the remarks, if it's incorrect (this is what I understood of it ;) ), feel free to edit.

Reply:
Well I think it should be removed in NLog 5.0. So better to obsolete it now, so people are warned.

Reply:
Well I don't like to commit on top of your commits. Try to back away when you start committing into my branches, because of my force-commit-style :)

Reply:
I'm a bit confused. Working in one (feature) branch with multiple people works flawless. Why can't we push another commit here? It works very well and especially when pulled rebase is set on both sides (`git config --global pull.rebase true`)

With my commits I'm trying to help here - not obstructing commiting ;)

Reply:
I'm not used to receive commits as feedback on my tasks. Prefer comments on the overall idea or comments on the individual code lines.

Reply:
It's sometimes hard to explain a refactoring ;) (see in the past)

Reply:
anyway, text updated